editor/svg-editor.js

```diff
@@ -1049,8 +1049,13 @@ TODOS
 					return;
 				}
 
-				// Opens the SVG in new window
-				var win = wind.open('data:image/svg+xml;base64,' + Utils.encode64(svg));
+				// Since saving SVGs by opening a new window was removed in Chrome use artificial link-click
+				// https://stackoverflow.com/questions/45603201/window-is-not-allowed-to-navigate-top-frame-navigations-to-data-urls
+				var a  = document.createElement('a');
+				a.href = 'data:image/svg+xml;base64,' + Utils.encode64(svg);
+				a.download = 'icon.svg';
```

Comment:
Ideally this should use the title of the document (.svg), but I think this is good enough for now.